testcases/OpTestHeartbeat.py

```diff
@@ -30,72 +30,21 @@
-from common.OpTestHost import OpTestHost
-from common.OpTestUtil import OpTestUtil
-from common.OpTestSystem import OpTestSystem
-
-
-class OpTestHeartbeat():
-    ##  Initialize this object
-    #  @param i_bmcIP The IP address of the BMC
-    #  @param i_bmcUser The userid to log into the BMC with
-    #  @param i_bmcPasswd The password of the userid to log into the BMC with
-    #  @param i_bmcUserIpmi The userid to issue the BMC IPMI commands with
-    #  @param i_bmcPasswdIpmi The password of BMC IPMI userid
-    #  @param i_ffdcDir Optional param to indicate where to write FFDC
-    #
-    # "Only required for inband tests" else Default = None
-    # @param i_hostIP The IP address of the HOST
-    # @param i_hostuser The userid to log into the HOST
-    # @param i_hostPasswd The password of the userid to log into the HOST with
-    #
-    def __init__(self, i_bmcIP, i_bmcUser, i_bmcPasswd,
-                 i_bmcUserIpmi, i_bmcPasswdIpmi, i_ffdcDir=None, i_hostip=None,
-                 i_hostuser=None, i_hostPasswd=None):
-        self.cv_BMC = OpTestBMC(i_bmcIP, i_bmcUser, i_bmcPasswd, i_ffdcDir)
-        self.cv_IPMI = OpTestIPMI(i_bmcIP, i_bmcUserIpmi, i_bmcPasswdIpmi,
-                                  i_ffdcDir)
-        self.cv_HOST = OpTestHost(i_hostip, i_hostuser, i_hostPasswd, i_bmcIP)
-        self.cv_SYSTEM = OpTestSystem(i_bmcIP, i_bmcUser, i_bmcPasswd,
-                         i_bmcUserIpmi, i_bmcPasswdIpmi, i_ffdcDir, i_hostip,
-                         i_hostuser, i_hostPasswd)
-        self.util = OpTestUtil()
-
-    ##
-    # @brief This function will cover following test steps
-    #        1. It will check for os level and get kernel version
-    #        2. It will check for existence of ps command
-    #        3. It will check for kopald service is running 
-    #
-    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
-    #
-    def test_kopald_service(self):
-        self.cv_SYSTEM.sys_bmc_power_on_validate_host()
-
-        # Get OS level
-        l_oslevel = self.cv_HOST.host_get_OS_Level()
-
-        # Get kernel version
-        l_kernel = self.cv_HOST.host_get_kernel_version()
-
-        # Checking for ps command 
-        self.cv_HOST.host_check_command("ps")
-
-        l_cmd = "ps -ef | grep -i kopald"
-        print l_cmd
-        l_res = self.cv_HOST.host_run_command(l_cmd)
-        print l_res
-        if (l_res.__contains__('[kopald]')):
-            return BMC_CONST.FW_SUCCESS 
-        else:
-            l_msg = "kopald service is not running"
-            print l_msg
-            raise OpTestError(l_msg)
+import unittest
+
+import OpTestConfiguration
+from common.OpTestSystem import OpSystemState
+
+class OpTestHeartbeat(unittest.TestCase):
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.cv_IPMI = conf.ipmi()
+        self.cv_SYSTEM = conf.system()
+
+    def runTest(self):
+        self.cv_SYSTEM.goto_state(OpSystemState.PETITBOOT_SHELL)
```

Comment:
a2d541365dca0dfa3a8cf16217dda223df9e95e8 and 27aaba0d68fc56cabe2b8e5b9a4acd2e80fbf2af implement this.

Had some fun with Ubuntu grep aliases